tests/wpt/meta/IndexedDB/idb-explicit-commit-throw.any.js.ini

```diff
@@ -1,8 +1,10 @@
 [idb-explicit-commit-throw.any.html]
+  expected: TIMEOUT
   [Any errors in callbacks that run after an explicit commit will not stop the commit from being processed.]
-    expected: FAIL
```

Comment:
Anything that requires `create_index` and has an underlying bug is now timing out.